kubos/utils/git_utils.py

```diff
@@ -32,23 +32,45 @@ def get_tag_list(repo):
     tags = repo.tags
     tag_list = []
     for tag in tags:
-        tag_list.append(tag)
+        tag_list.insert(0, tag.name)
     return tag_list
 
 
-def print_tag_list(tag_list):
+def filter_minor_tags(display_num, tag_list):
```

Comment:
Should we use a term other than "minor version", as that refers to the 2nd number in the version specifier. Maybe "build version" or "build number"?

Also as far as the new tag formatting goes, if we want to continue down the path of following semver, http://semver.org/, which is what `yotta` specifies, it looks like they have a specified way to indicate build versions `<major>.<minor>.<patch>+<buildNumber>`.